### PR TITLE
Add development setup instructions

### LINKED ISSUE #290: Add development setup instructions
Today, we don't have good instructions on how to get set up with a development environment for tf-encrypted.

### PR DESCRIPTION
**Pull request by ianlivingstone**
Closes #290

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+Once the clone is complete, you can switch into the tf-encrypted library using `cd tf-encrypted` and then install the dependencies using `make bootstrap` which will error if any dependencies are missing.
+
+```
+$ cd tf-encrypted
+$ make bootstrap
+```
+
+**Begin Developing**
+
+You should now be able to begin developing! You can check that your setup works by running our tests via `make tests`.
```
Comment (justin1121) [inline .github/CONTRIBUTING.md]: Think this should be `make test`.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+consumers of this project the corresponding documentation should be updated so
+users always have up to date documentation that reflects the true behaviour of
+the library.
+
+You can build the project locally using the `make docs` command which will
+subsequently output the html version of our docs to your `build` folder. You
+can view the docs after their built using your browser by running the command
+`open build/html/index.html`.
+
+## Typing
```
Comment (justin1121) [inline .github/CONTRIBUTING.md]: Wonder if we should remove this typing stuff?

Comment (ianlivingstone) [reply .github/CONTRIBUTING.md]: I think so - but we should do it in all one move.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+Looking to contribute to tf-encrypted but not sure what to tackle? We've labelled all of the ready issues with [help wanted](https://github.com/mortendahl/tf-encrypted/labels/help%20wanted). For those just looking to get their feet wet we've labelled bite sized issues with [good first issue](https://github.com/mortendahl/tf-encrypted/labels/good%20first%20issue).
+
+Have an idea but an issue doesn't exist or it's not labelled with `help wanted`? No problem, feel free to kick off the conversation by suggesting a possible solution or by opening a pull request!
+
+If you ever need help, don't hesitate to ask!
+
+# Reporting a bug
+
+Think you've found a bug? Let us know by opening na [issue in our tracker](https://github.com/mortendahl/tf-encrypted/issues) and apply the "bug" label!
```
Comment (justin1121) [inline .github/CONTRIBUTING.md]: na -> an

**Pull request merged**